sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -65,56 +68,54 @@ or [Azure CLI][azure_cli_endpoint_lookup]:
-    from azure.core.credentials import AzureKeyCredential
-
-    endpoint = "https://<region>.api.cognitive.microsoft.com/"
-    credential = AzureKeyCredential("<api_key>")
-    form_recognizer_client = FormRecognizerClient(endpoint, credential)
-    ```
-
-2. To use an [Azure Active Directory (AAD) token credential][cognitive_authentication_aad],
-   provide an instance of the desired credential type obtained from the
-   [azure-identity][azure_identity_credentials] library.
-   Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain]
-   name for your resource in order to use this type of authentication.
-
-   Authentication with AAD requires some initial setup:
-   * [Install azure-identity][install_azure_identity]
-   * [Register a new AAD application][register_aad_app]
-   * [Grant access][grant_role_access] to Form Recognizer by assigning the `"Cognitive Services User"` role to your service principal.
-
-   After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
-   As an example, [DefaultAzureCredential][default_azure_credential]
-   can be used to authenticate the client:
-
-   Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
-   `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
-
-   Use the returned token credential to authenticate the client:
-    ```python
-    from azure.identity import DefaultAzureCredential
-    from azure.ai.formrecognizer import FormRecognizerClient
-    token_credential = DefaultAzureCredential()
-
-    form_recognizer_client = FormRecognizerClient(
-        endpoint="https://<my-custom-subdomain>.cognitiveservices.azure.com/",
-        credential=token_credential
-    )
-    ```
+#### Get the API key
+
+The API key can be found in the Azure Portal or by running the following Azure CLI command:
+
+```az cognitiveservices account keys list --name "resource-name" --resource-group "resource-group-name"```
+
+#### Create the client with AzureKeyCredential
+
+To use an [API key][cognitive_authentication_api_key] as the `credential` parameter,
+pass the key as a string into an instance of [AzureKeyCredential][azure-key-credential].
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.formrecognizer import FormRecognizerClient
+
+endpoint = "https://<region>.api.cognitive.microsoft.com/"
+credential = AzureKeyCredential("<api_key>")
+form_recognizer_client = FormRecognizerClient(endpoint, credential)
+```
+
+#### Create the client with an Azure Active Directory credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also 
+authenticate with Azure Active Directory using the [azure-identity][azure_identity] library.
+Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain]
```

Comment:
>Note that regional endpoints do not support AAD authentication.

Is this line still valid?

Reply:
yes, https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-azure-active-directory

I decided to try it just to make sure and got this error message:
`(BadRequest) Please provide a custom subdomain for token authentication, otherwise API key is required. (Please see https://aka.ms/cogsvc-authenticatewithtoken)`

Reply:
FYI (more out of curiosity) the error message here was improved thanks to Jon Gallant trying out library and then Chris Stone helping spreading the word,
See issue https://github.com/Azure/azure-sdk-for-net/issues/12813